README.md

```diff
@@ -89,7 +96,16 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An Entity Relational Diagram is a way for us to model our database entities such
+that we can describe what kind of attributes each entity should have, and its
+relationship with the surrounding entities. It's a way to plan and structure database
+tables in an application. For the four models given, we can probably say that a Lamp
+ has a one to many relationship with a Genie; a Lamp has one Genie inhabiting it at
+  any given time, but multiple Genies may have belonged to that Lamp (at least in
+    the case of Alladin - only one Genie can inhabit a lamp at a time..iirc). A person
+     to pet relationship can go either way. A person can have one pet and that pet belongs
+      to a specific person. A person might also have multiple pets and those pets all
+      belong to same owner; that can be a one to many relationship between person and pet.
 ```
```

Comment:
I hadn't thought about the Genie-to-person relationship that way! Thinking of "has" in terms of the full history of the lamp is, indeed, a one-to-many relationship, whereas thinking only of the current moment in time, it would be one-to-one. Your example really illustrates the framing choices that you as a developer get to make in designing your ERD.